package.json

```diff
@@ -51,6 +50,12 @@
   },
   "dependencies": {
     "aws-sdk": "~2.2.28",
+    "babel-cli": "^6.24.1",
```

Comment:
I did a quick audit and I'm going to prune down the dependencies we're shipping. Per https://babeljs.io/docs/plugins/transform-runtime/:

>NOTE - Production vs. development dependencies

> In most cases, you should install babel-plugin-transform-runtime as a development dependency (with --save-dev) and babel-runtime as a production dependency (with --save).

> The transformation plugin is typically used only in development, but the runtime itself will be depended on by your deployed/published code. See the examples below for more details.

Moving the `babel-*` packages to `devDependencies` and adding `babel-runtime` to regular `dependencies` cut the extracted size from 70mb to 30mb and the package size from 8.8mb to 4.2mb.